aqua.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# Copyright 2025 Ian Lewis
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/aqua-yaml.json
+# aqua - Declarative CLI Version Manager
+# https://aquaproj.github.io/
+checksum:
+  enabled: true
+  require_checksum: true
+  supported_envs:
+    - all
+registries:
+  - type: standard
+    ref: v4.335.0 # renovate: depName=aquaproj/aqua-registry
+packages:
+  - name: rhysd/actionlint@v1.7.7
+  - name: koalaman/shellcheck@v0.10.0
+  - name: jqlang/jq@jq-1.7.1
+  - name: golangci/golangci-lint@v2.1.2
```

Comment:
golangci-lint should be v1 rather than v2. v2 upgrade with happen as part of #49